src/platform/user/profile/vet360/containers/ReceiveTextMessages.jsx

```diff
@@ -133,7 +134,8 @@ export function mapStateToProps(state, ownProps) {
     hasError ||
     isPending;
   const transactionSuccess =
-    state.vet360.transactionStatus === 'COMPLETED_SUCCESS';
+    state.vet360.transactionStatus ===
+    VET360.TRANSACTION_STATUS.COMPLETED_SUCCESS;
```

Comment:
Maybe this branch has unrelated changes in it?

Reply:
I was just correcting some magic strings that were pointed out in the last pull request.